packages/jest-environment-node/src/index.ts

```diff
@@ -60,7 +54,8 @@ class NodeEnvironment {
       },
     });
 
-    const timerRefToId = (timer: Timer): ?number => (timer && timer.id) || null;
+    const timerRefToId = (timer: Timer): number | undefined =>
+      (timer && timer.id) || undefined;
```

Comment:
Please note that I've changed the possible return type from null to undefined here. FakeTimers does not like the timerConfig if timerRefToId returns null (but undefined is OK).

Reply:
That's fine. Happy to get rid of the `null`!